scripts/nn/layers/softmax2d_loss.dml

```diff
@@ -0,0 +1,80 @@
+#-------------------------------------------------------------
```

Comment:
we already have the softmax2d and the associated cross_entropy_loss2d layers in the `nn` library.  let's have Caffe2DML simply use those two layers, rather than adding this wrapper layer.